src/test/Components/CreateImageWizard/CreateImageWizard.test.tsx

```diff
@@ -88,419 +82,6 @@ describe('Create Image Wizard', () => {
-    await clickNext();
-    await clickNext();
-    await clickNext();
-    await enterBlueprintName();
-    await clickNext();
-    // informational modal pops up in the first test only as it's tied
-    // to a 'imageBuilder.saveAndBuildModalSeen' variable in localStorage
-    await openAndDismissSaveAndBuildModal();
-
-    const createBlueprintBtn = await screen.findByRole('button', {
-      name: /Create blueprint/,
-    });
-
-    user.click(createBlueprintBtn);
-
-    // returns back to the landing page
-    await waitFor(() =>
-      expect(router?.state.location.pathname).toBe('/insights/image-builder')
-    );
-    // set test timeout of 10 seconds
-  }, 10000);
-});
-
-describe('Step Upload to Google', () => {
-  beforeEach(() => {
-    vi.clearAllMocks();
-    router = undefined;
-  });
-
-  const user = userEvent.setup();
-  const setUp = async () => {
-    ({ router } = await renderCustomRoutesWithReduxRouter(
-      'imagewizard',
-      {},
-      routes
-    ));
-
-    // select gcp as upload destination
-    const uploadGcp = await screen.findByTestId('upload-google');
-
-    user.click(uploadGcp);
-
-    await clickNext();
-
-    await screen.findByRole('heading', {
-      name: 'Target environment - Google Cloud Platform',
-    });
-  };
-
-  test('clicking Next loads Registration', async () => {
-    await setUp();
-
-    const shareRadioButton = await screen.findByText(
-      /share image with a google account/i
-    );
-    user.click(shareRadioButton);
-
-    const googleEmailInput = await screen.findByTestId('principal');
-
-    await waitFor(() => user.type(googleEmailInput, 'test@test.com'));
-    await clickNext();
-
-    await screen.findByRole('textbox', {
-      name: 'Select activation key',
-    });
-
-    await screen.findByText(
-      'Automatically register and enable advanced capabilities'
-    );
-  });
-
-  test('clicking Back loads Release', async () => {
-    await setUp();
-
-    await clickBack();
-
-    await screen.findByTestId('upload-google');
-  });
-
-  test('clicking Cancel loads landing page', async () => {
-    await setUp();
-
-    await verifyCancelButton(router);
-  });
-
-  test('the google account id field is shown and required', async () => {
-    await setUp();
-
-    const principalInput = await screen.findByTestId('principal');
-    expect(principalInput).toHaveValue('');
-    expect(principalInput).toBeEnabled();
-  });
-
-  test('the google email field must be a valid email', async () => {
-    await setUp();
-
-    await waitFor(async () =>
-      user.type(await screen.findByTestId('principal'), 'a')
-    );
-    expect(await getNextButton()).toHaveClass('pf-m-disabled');
-    expect(await getNextButton()).toBeDisabled();
-    await waitFor(async () =>
-      user.type(await screen.findByTestId('principal'), 'test@test.com')
-    );
-    expect(await getNextButton()).not.toHaveClass('pf-m-disabled');
-    expect(await getNextButton()).toBeEnabled();
-  });
-});
```

Comment:
And this to the GCP test file? `src/test/Components/CreateImageWizard/steps/TargetEnvironment/GCPTarget.test.tsx`